README.md

```diff
@@ -176,6 +176,18 @@ i.e if your original database is named `gis`, you can restore it into a new data
  docker-compose exec dbbackups /backup-scripts/restore.sh
  ```
 
+## Restoring from S3 bucket
+The script uses [s3cmd](https://s3tools.org/s3cmd) for restoring files S3 bucket to a postgresql database.
+
+To restore from S3 bucket, first you have to exec into your running container. You have to launch the /backup-scripts/restore.sh with two parameters 
```

Comment:
Would it be possible to use env variable to restore the DB, rather than exec into the container ?

Reply:
Hey @NyakudyaA, I think it's better to exec into the container because the image already have the double functionnality (backup and restore), I find it easier than launching another container with other variables just for the sake of restoring a database. If you have a container or a pod already running for backup it's better to use it to restore at the same time. But if you really want it to be separated I understand.